### ISSUE TITLE
How to fine-tune the bart-large-mlni model on a classification task?

### ISSUE THREAD

**Issue by fmplaza** [pytorch/fairseq#3700]
Hi,

I'm trying to fine-tune the bart-large-mnli model on an emotion classification dataset, but the accuracy significantly drops after running my code. I'm doing something wrong while fine-tuning the model, could anyone help me?

Code: https://drive.google.com/file/d/1jvUjQW-3KJOKNdNzdM9fQpvraqRlLphp/view?usp=sharing

Thanks a lot!
